### ISSUE TITLE
String printing

### ISSUE THREAD

**Issue by valerionew** [5N44P/ht1621-7-seg#11]
Since a 7 segment display can print non-number characters, a `print(char*symbols)` or a `print(String symbols)` should be implemented.
Possible symbols to be printed are:
- A
- b
- c and C
- d
- e
- f / F
- g
- h and H
- I 
- J
- L
- n (maybe?)
- o and O
- P
- q
- r
- S
- t
- u and U
- Y (maybe?) 
- °
- \- (minus)
- _ (underscore)
- | (pipe?)
- "
- =

Missing alphabet characters: K, M, V, W, X, (Z?)

**Comment by pk** [comment]
Hi Valerio, I think I've started with the implementation, so I'd like to take this one. I make the fork and we can discuss the solution.

**Comment by valerionew** [comment]
Sure, thanks! Let me know if you need anything

**Comment by pk** [comment]
Hi Valerio, have a look at the implementation (I'm not that proficient in C) of the character dipsplay. I've created also KitchenSink sketch to have sll options rendered.

I've implemented rendering of the characters above as well as the numbers. So I can easily print things like 123(deg symbol) and so on.

Let me know what you think, I'm not sure how to better implement Character -> Seg map than what I did.

https://github.com/pk/ht1621-7-seg
